pkg/core/blockchain.go

```diff
@@ -591,7 +593,9 @@ func (bc *Blockchain) GetAccountState(scriptHash util.Uint160) *AccountState {
 	var as *AccountState
 	bc.Store.Seek(storage.STAccount.Bytes(), func(k, v []byte) {
 		var a AccountState
-		if err := a.DecodeBinary(io.NewBinReaderFromBuf(v)); err == nil && a.ScriptHash == scriptHash {
+		r := io.NewBinReaderFromBuf(v)
+		a.DecodeBinary(r)
```

Comment:
hard to dive deeper there. But got a q: Won't DecodeBinary panic in case of r == nil ?

Reply:
It will. That's why NewBin* thingies never return `nil`

Reply:
kk, I see